react/styles.css

```diff
@@ -1,8 +1,8 @@
 .product-identifier {
 }
-.product-identifier__label {
+.product-identifierLabel {
 }
-.product-identifier__separator {
+.product-identifierSeparator {
 }
-.product-identifier__value {
+.product-identifierValue {
```

Comment:
This renaming would result in a breaking change for anyone that is using this component.
These classes are the ones a user would customize in their theme, via `vtex.product-identifier.css`. 
If a user was already using this app, they might have already written their `vtex.product-identifier.css` file, using the previous CSS handles. This renaming would break their customization.

Reply:
Also, this file is no longer needed, so you could just delete it :)

Reply:
I just deleted the file as you recommended! Thank's for the tip